src/components/backend-ai-resource-policy-list.ts

```diff
@@ -538,25 +584,31 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
 
   _modifyResourcePolicy() {
     let name = this.shadowRoot.querySelector('#id_new_policy_name').value;
-    let input = this._readResourcePolicyInput();
-
-    window.backendaiclient.resourcePolicy.mutate(name, input)
-      .then(({modify_keypair_resource_policy}) => {
-        if (modify_keypair_resource_policy.ok) {
+    try {
+      let input = this._readResourcePolicyInput();
+
+      window.backendaiclient.resourcePolicy.mutate(name, input)
+        .then(({modify_keypair_resource_policy}) => {
+          if (modify_keypair_resource_policy.ok) {
+            this.shadowRoot.querySelector('#modify-policy-dialog').hide();
+            this.notification.text = "Resource policy successfully updated.";
+            this.notification.show();
+            this.refresh();
+          }
+        }).catch(err => {
+        console.log(err);
```

Comment:
`.then` in line 591 is placed in new line, so it would be better if `.catch` is placed in new line, either. `console.log(err)` and the subsequent block contents should be indented one-level more.